### PR TITLE
Add instructions to generate the documentation locally

### PR DESCRIPTION
**Pull request by lesteve**

### CODE REVIEW

**Comment by mdiazmel** [comment]
I don't know if it matters but maybe we can also remember to the contributors that instead of building locally a built version of each PR is available in `https://jean-zay-doc--[#PR].org.readthedocs.build/en/[#PR]/`. For example, for this PR, the corresponding version of the documentation is in https://jean-zay-doc--77.org.readthedocs.build/en/77/

Up to you if you want to include this information, otherwise we can merge it.

**Comment by lesteve** [comment]
Done, to see how it is rendered by github: https://github.com/jean-zay-users/jean-zay-doc/blob/f0f275ddfbeaafa0e300e8ea9e5c55885c38939b/README.md

File: README.md [hunk 1]
```diff
@@ -24,3 +24,37 @@ better a [Pull Request (PR)](https://github.com/jean-zay-users/jean-zay-doc/comp
...
 [there](https://gitter.im/jean-zay-users/jean-zay-doc) and ask questions!
+
+# See the generated doc in a PR
+
+You can see the generated doc in a PR at the bottom of the PR page. This allows
+you to double-check that the rendered HTML looks the way you intended.
+
+It will look like the screenshot below. To view the generated doc you can click
+on the "Details" link:
+![](https://user-images.githubusercontent.com/1680079/146903507-d56addb6-c038-4d01-a151-933a8f261af2.png)
```
Comment (lesteve) [inline README.md:35]: This is a bit of a quick and dirty hack. I did a snapshot and pasted it in a github comment, this way github creates a URL where the image can be accessed.

I was a bit too lazy to add the image to the repo, but I can do this if you insist :wink:

Comment (mdiazmel) [reply README.md:35]: I didn't know that this was possible! (thanks for the hack 😉 ).

Let's merge it!

**Pull request merged**